aten/src/ATen/native/cuda/UpSampleBilinear2d.cu

```diff
@@ -179,7 +184,7 @@ static void upsample_bilinear2d_out_cuda_template(
         using accscalar_t = at::acc_type<scalar_t, true>;
 
         auto idata = input.packed_accessor64<scalar_t, 4>();
-        auto odata = output.packed_accessor64<scalar_t, 4>();
+        auto odata = output_c.packed_accessor64<scalar_t, 4>();
```

Comment:
It looks like `packed_accessor` correctly handles the strides of the tensor, and bilinear/trilinear interpolation produce correct results even without this fix, so maybe undo these changes?
Difference of 1e-4 looks kinda large, but maybe that's and in any case it's not changed by this